docs/en/extending-doris/flink-doris-connector.md

```diff
@@ -132,71 +132,86 @@ INSERT INTO flink_doris_sink select name,age,price,sale from flink_doris_source
+
+
+* Json data
+
 ```java
-// -------- sink with raw json string stream --------
 Properties pro = new Properties();
-pro.setProperty("format", "json");
-pro.setProperty("strip_outer_array", "true");
-env.fromElements( "{\"longitude\": \"116.405419\", \"city\": \"北京\", \"latitude\": \"39.916927\"}")
-     .addSink(
-     	DorisSink.sink(
-            DorisReadOptions.builder().build(),
-         	DorisExecutionOptions.builder()
-                    .setBatchSize(3)
-                    .setBatchIntervalMs(0l)
-                    .setMaxRetries(3)
-                    .setStreamLoadProp(pro).build(),
-         	DorisOptions.builder()
-                    .setFenodes("FE_IP:8030")
-                    .setTableIdentifier("db.table")
-                    .setUsername("root")
-                    .setPassword("").build()
-     	));
-
-OR
-env.fromElements("{\"longitude\": \"116.405419\", \"city\": \"北京\", \"latitude\": \"39.916927\"}")
-    .addSink(
-    	DorisSink.sink(
-        	DorisOptions.builder()
-                    .setFenodes("FE_IP:8030")
-                    .setTableIdentifier("db.table")
-                    .setUsername("root")
-                    .setPassword("").build()
-    	));
-
-
-// -------- sink with RowData stream --------
-DataStream<RowData> source = env.fromElements("")
-    .map(new MapFunction<String, RowData>() {
-        @Override
-        public RowData map(String value) throws Exception {
-            GenericRowData genericRowData = new GenericRowData(3);
-            genericRowData.setField(0, StringData.fromString("北京"));
-            genericRowData.setField(1, 116.405419);
-            genericRowData.setField(2, 39.916927);
-            return genericRowData;
-        }
-    });
+env.fromElements(
+                "{\"longitude\": \"116.405419\", \"city\": \"北京\", \"latitude\": \"39.916927\"}",
+                "{\"longitude\": \"116.405419\", \"city\": \"beijing\", \"latitude\": \"39.916927\"}"
+        )
+        .addSink(
+                DorisSink.sink(
+                        DorisReadOptions.builder()
+                                .build(),
+                        DorisExecutionOptions.builder()
+                                .setBatchSize(3)
+                                .setBatchIntervalMs(0l)
+                                .setMaxRetries(3)
+                                .setStreamLoadProp(pro)
+                                .setDataFormat(DorisExecutionOptions.DataFormat.JSON)
+                                .setStripOuterArray(true)
+                                .setMergeType(DorisExecutionOptions.MergeType.MERGE)
+                                .setDeleteLabel("city='beijing'")
+                                .build(),
+                        DorisOptions.builder()
```

Comment:
Agreed. I think following 3 params can be removed from example:
```
                                .setStripOuterArray(true)
                                .setMergeType(DorisExecutionOptions.MergeType.MERGE)
                                .setDeleteLabel("city='beijing'")
```

And the newly added params should be added to `通用配置项`

Reply:
`通用配置项` means `setStreamLoadProp()`?

Reply:
No. Here in the document:

![Uploading image.png…]()

Reply:
No. Here in the document:

![Uploading image.png…]()

Reply:
http://doris.incubator.apache.org/master/zh-CN/extending-doris/flink-doris-connector.html#%E9%85%8D%E7%BD%AE   

search : 通用配置

Reply:
`通用配置` is a good place to add newly params in flink table api(sql), in the `DataStreamSink` it may be different. 
i am finding out a better way to add them.
the current version puts the params into Properties by `setStreamLoadProp()`, it could be a better way , if the user is familiar with the stream load and json load. But if the user is not familiar,  is there a good way?

Reply:
Oh I see, could you please contact me with Wechat(morningman-cmy). There are some details need to be discussed.